Revert bug 1497976 as close_fds on Python 2 is too slow

diff --git build/moz.configure/util.configure build/moz.configure/util.configure
index 25862fee7c32..3fc725a5124e 100644
--- build/moz.configure/util.configure
+++ build/moz.configure/util.configure
@@ -22,7 +22,6 @@
 
 # A wrapper to obtain a process' output and return code.
 # Returns a tuple (retcode, stdout, stderr).
-@imports('os')
 @imports(_from='__builtin__', _import='unicode')
 @imports('subprocess')
 @imports(_from='mozbuild.shellutil', _import='quote')
@@ -45,13 +44,7 @@
 
     log.debug('Executing: `%s`', quote(*args))
     proc = subprocess.Popen(args, stdout=subprocess.PIPE,
-                            stderr=subprocess.PIPE,
-                            # On Python 2 on Windows, close_fds prevents the
-                            # process from inheriting stdout/stderr.
-                            # Elsewhere, it simply prevents it from inheriting
-                            # extra file descriptors, which is what we want.
-                            close_fds=os.name != 'nt',
-                            **kwargs)
+                            stderr=subprocess.PIPE, **kwargs)
     stdout, stderr = proc.communicate()
     return proc.wait(), stdout, stderr
 
